js/script.js

```diff
@@ -0,0 +1,103 @@
+const openbtn = document.querySelector('.open');
+const closebtn = document.querySelector('.close');
+const menu = document.querySelector('.curtain');
+const links = document.querySelectorAll('.menu-link');
+openbtn.addEventListener('click', () => {
+  menu.style.height = '100%';
+  document.body.style.overflow = 'hidden';
+});
+closebtn.addEventListener('click', () => {
+  menu.style.height = '0';
+  document.body.style.overflow = 'visible';
+});
+
+links.forEach((link) => {
+  link.addEventListener('click', () => {
+    menu.style.height = '0';
+    document.body.style.overflow = 'visible';
+  });
+});
+
+const houses = [{
+  image: '../images/h1.jpg',
+  title: 'Luzury',
+  discription: 'We are buiding luzury houses in Malindi. The first market release will be on as soom as Novermber 2023.',
+  author: 'Petro Lesapiti',
+
+},
+{
+  image: '../images/h2.jpg',
+  title: 'Live',
+  discription: 'We are buiding luzury houses in Malindi. The first market release will be on as soom as Novermber 2023.',
+  author: 'Petro Lesapiti',
+},
+{
+  image: '../images/h3.jpg',
+  title: 'Parking',
+  discription: 'We have ample parking space in all our highend properties in Keleleshwa, Kilimani and Eastlands.',
+  author: 'Petro Lesapiti',
+
+},
+{
+  image: '../images/h1.jpg',
+  title: 'Money',
+  discription: 'Partnered with all major banks for our customers who are looking to secure morgage facilities.',
+  author: 'Petro Lesapiti',
+},
+];
```

Comment:
Kindly add two more elements to this section, remember we have to match the given design as much as possible 😄 
In the given design there are sex elements, you only have four:
![image](https://user-images.githubusercontent.com/87448628/181751238-9265ed61-9280-49db-8a10-25ed4bb927a2.png)